docker-compose.yml

```
@@ -1,20 +1,41 @@
----
+version: '3.8'
 services:
-  postgres:
+  db:
     image: postgres:14
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
     environment:
-      POSTGRES_USER: ${DATABASE_USERNAME}
-      POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
-      POSTGRES_DB: pet_rescue_development
+      POSTGRES_DB: myapp_development
```

```suggestion
      POSTGRES_DB: petrescue_development
```

`myapp` screams demoware.  :laughing: